CloudinaryManager.js

```diff
@@ -0,0 +1,58 @@
+export default class CloudinaryManager {
+    constructor(options) {
+        this.options = options;
+    }
+
+    createMediaString(formdata) {
+        var mediaStr;
+
+        if (formdata.minWidth !== "") {
+            mediaStr = `media="(min-width: ${formdata.minWidth})"`;
+        } else if (formdata.maxWidth !== "") {
+            mediaStr = `media="(max-width: ${formdata.maxWidth})"`;
+        }
+
+        return mediaStr;
+    }
+
+    createImageString(formdata) {
+        let acctName = "dnginndga";
+        var queryArr = [];
+
+        if (formdata.width !== "") {
+            queryArr.push("w_" + formdata.width + "px");
+        }
+
+        if (formdata.height !== "") {
+            queryArr.push("h_" + formdata.height + "px");
+        }
+
+        if (formdata.gravity != "") {
+            queryArr.push("g_" + formdata.gravity);
+        }
+
+        if (formdata.crop != "") {
+            queryArr.push("c_" + formdata.crop);
+        }
+
+        var imgStr;
+        var queryStr;
+
```

Comment:
Can make this simpler. 

```suggestion
return queryArr.length == 0 
  ? `https://res.cloudinary.com/${this.options.acctName}/image/upload/${formData.codename}
  : `https://res.cloudinary.com/${this.options.acctName}/image/upload/${queryArr.join(",")}/${formData.codename}
```

Reply:
Can remove the imgStr and queryStr variables also.

Reply:
done